src/app/services/api.ts

```diff
@@ -0,0 +1,18 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+const localData = [
+  { id: 1, n: "Alan Wake", c: ["TPS", "Adventure", "Horror"] },
+  { id: 2, n: "Jason Bourne", c: ["Movie", "Thriller", "Spy"] },
+  { id: 3, n: "Bruce Wayne", c: ["Batman", "Philanthropist", "Orphan"] },
+];
+
+export const api = createApi({
+  baseQuery: fetchBaseQuery({ baseUrl: '/' }),
+  endpoints: (builder) => ({
+    getItems: builder.query<any[], void>({
```

Comment:
I feel it would be beneficial to specify the return type here instead of using `any`. Is there anything that caused difficulties here?